src/components/grid-aware/StatsBlock/StatsBlock.module.css

```diff
@@ -73,6 +74,17 @@
   margin: 0;
 }
 
+.impactTitle {
+  color: var(--color-sheltertech-blue);
+}
+
+.subtitle {
+  font: var(--font-body-medium);
+  grid-column: 2 / span 10;
```

Comment:
Looking at the Figma designs, I think this only spans 8 columns, not ten, since when you turn on the layout grid in Figma, you can see that the subtitles wrap at the 9th column.